FE Build failure by cesium

## Description
After npm installl and first frontend start fail building on module `@cesium`

after quickly investigations one solution can be upgrade in `package.json`
```diff
- "cesium": "1.131.0",
+ "cesium": "1.133.1",
```

## How to reproduce
```
npm install
npm run fe:start
```
fail at build of with this message:
```
modules by path ./lib/*./lib/locales (ignored) 30 bytes                                                                                                                                                                           
  ./lib/locales (ignored) 15 bytes [built] [code generated]                                                                                                                                                                       
  ./lib/locales (ignored) 15 bytes [built] [code generated]                                                                                                                                                                       
+ 18 modules                                                                                                                                                                                                                      
                                                                                                                                                                                                                                  
ERROR in ./node_modules/@cesium/engine/Source/DataSources/KmlDataSource.js 45:0-59                                                                                                                                                
Module not found: Error: Package path ./lib/zip-no-worker.js is not exported from package /home/user/MapStore2/node_modules/@zip.js/zip.js (see exports field in /home/user/MapStore2/node_modules/@zip.js/zip.js/package.json)   
 @ ./node_modules/@cesium/engine/index.js 92:0-81 92:0-81                                                                                                                                                                         
 @ ./node_modules/cesium/Source/Cesium.js 2:0-51 2:0-51 3:0-40 3:0-40 4:0-45 4:0-45 5:0-
.........
```

*Expected Result*
build and start webapp

*Current Result*
crash at build phase


- [x] Not browser related


## Other useful information
npm version 10.8.2
nodejs version 20.19.0

--- a/package.json
+++ b/package.json
@@ -149,7 +149,7 @@
     "bootstrap": "3.4.1",
     "buffer": "6.0.3",
     "canvas-to-blob": "0.0.0",
-    "cesium": "1.131.0",
+    "cesium": "1.133.1",
     "chroma-js": "1.3.7",
     "classnames": "2.2.5",
     "codemirror": "5.65.16",

